reaktive/src/jsMain/kotlin/com/badoo/reaktive/timer/Timers.kt

```diff
@@ -0,0 +1,21 @@
+package com.badoo.reaktive.timer
+
+import kotlin.browser.window
+import com.badoo.reaktive.timer.external.clearInterval as clearIntervalExternal
+import com.badoo.reaktive.timer.external.clearTimeout as clearTimeoutExternal
+import com.badoo.reaktive.timer.external.setInterval as setIntervalExternal
+import com.badoo.reaktive.timer.external.setTimeout as setTimeoutExternal
+
+private val isWindowDefined: Boolean = jsTypeOf(window) != "undefined"
```

Comment:
So if this solutions works I'm fine with it. Can you write some tests? You could just write 4 tests each calling single method. Without any verification, just to make sure they do not crash. They will be run automatically for both browser and NodeJs.

Reply:
Removed the whole file, resolving